src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -12,11 +12,14 @@ import RouteModel from '../navigationLibrary/Route/RouteModel';
 import { airlineNameAndFleetHelper } from '../airline/airlineHelpers';
 import { convertStaticPositionToDynamic } from '../base/staticPositionToDynamicPositionHelper';
 import { speech_say } from '../speech';
+import { tau } from '../math/circle';
 import { abs } from '../math/core';
 import { distance2d } from '../math/distance';
-import { vlen } from '../math/vector';
-import { km } from '../utilities/unitConverters';
+import { calcTurnInitiationDistance } from '../math/flightMath';
+import { vlen, vradial, vsub } from '../math/vector';
+import { km, kn_ms, radiansToDegrees, degreesToRadians } from '../utilities/unitConverters';
```

Comment:
I looked this over and it looks like it happens after a merge I believe it was this merge d7fbc7bb94b9fc528512018f0d467c3181b0ef1b  and that reworked and changed some things in the aircraft controller